src/AutoBuild.jl

```diff
@@ -226,6 +259,7 @@ function build_tarballs(ARGS, src_name, src_version, sources, script,
         build_jll_package(src_name, build_version, code_dir, build_output_meta,
                           dependencies, bin_path; verbose=verbose,
                           extract_kwargs(kwargs, (:lazy_artifacts,))...,
+                          extract_kwargs(kwargs, (:usage_example,))...,
```

Comment:
Perhaps this would be better?
```suggestion
                          extract_kwargs(kwargs, (:lazy_artifacts, :usage_example))...,
```